icons/authentik.svg

```diff
@@ -0,0 +1,15 @@
+<svg width="1440" height="1024" fill="none" xmlns="http://www.w3.org/2000/svg"><title>Authentik</title>
+<rect width="1440" height="1024" viewBox="0 0 24 24" fill="white"/>
+<rect x="593" y="384" width="255" height="255" fill="#FD4B2D"/>
+<rect x="636" y="419" width="180" height="179" rx="40" fill="white"/>
+<rect x="689" y="554" width="73" height="82" fill="#FD4B2D"/>
+<rect x="636" y="496" width="104" height="29" fill="#FD4B2D"/>
+<rect x="689" y="461" width="54" height="64" fill="#FD4B2D"/>
+<rect x="701" y="452" width="10" height="21" fill="white"/>
+<rect x="720" y="447" width="10" height="21" fill="white"/>
+<ellipse cx="24.5851" cy="19.9267" rx="24.5851" ry="19.9267" transform="matrix(-0.0333544 0.999444 -0.996631 -0.0820221 642.359 485.269)" fill="white"/>
+<path d="M653.638 509.445C654.464 510.306 654.418 511.658 653.54 512.379L634.671 527.859C633.356 528.939 631.315 527.861 631.373 526.117L632.507 492.144C632.565 490.4 634.666 489.663 635.903 490.952L653.638 509.445Z" fill="white"/>
+<path d="M639.954 508.918C640.367 509.349 640.345 510.024 639.905 510.385L629.357 519.04C628.699 519.579 627.678 519.04 627.707 518.168L628.341 499.175C628.37 498.303 629.421 497.934 630.039 498.579L639.954 508.918Z" fill="#FD4B2D"/>
+<ellipse cx="13.4368" cy="10.8908" rx="13.4368" ry="10.8908" transform="matrix(-0.0333544 0.999444 -0.996631 -0.0820221 633.605 495.787)" fill="#FD4B2D"/>
+<rect x="593" y="384" width="255" height="255" fill="black" style="mix-blend-mode:saturation"/>
+</svg>
```

Comment:
There are some issues with this SVGs. I suggest you to read our [contributing guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md).

For example, you will need to compound every path into a single one, change the viewbox, remove colors and any extraneous attributes, ...

You can see if anything is wrong looking at the [actions](https://github.com/simple-icons/simple-icons/actions/runs/8091532777/job/22110739748?pr=10545).

You can also take a look at those [discussions](https://github.com/simple-icons/simple-icons/discussions/categories/help-with-svgs) if you need help!